src/frontend/src/flows/manage/deviceSettings.ts

```diff
@@ -36,53 +40,47 @@ export const deviceSettings = ({
 }): Setting[] => {
   const settings: Setting[] = [];
 
-  // Whether the device can be protected or not
-  if (shouldOfferToProtect(device)) {
-    settings.push({
-      label: "protect",
-      fn: () => protectDevice({ userNumber, connection, device, reload }),
-    });
-  }
-
-  // Whether the device can be unprotected or not
-  if (shouldOfferToUnprotect(device)) {
-    settings.push({
-      label: "unprotect",
-      fn: () =>
-        unprotectDevice(userNumber, connection, device, isOnlyDevice, reload),
-    });
+  // Whether the device can be (un)protected or not (only recovery phrases can be protected)
+  if (isRecoveryPhrase(device)) {
```

Comment:
I've inlined the "shouldOfferToProtect" checks because it wasn't clear what was happening